### PR TITLE
remove duplicate procedures

### PR DESCRIPTION
**Pull request by rosemarymarano**
In a Quick Start guide, it goes against technical writing best practices to implement duplicate procedures without a **known** benefit. (Please correct me if there is one.) These add to its maintenance and the likelihood of becoming out-of-sync. @ankedia advised me that copy & paste is easier for users, so I removed the procedural duplicate curl commands to download the files in two instances. From discussions with @tbarnes-us, I know he doesn't agree. **NOTE**: I kept the curl command to download the sample domain YAML file because there was no other way for users to view or modify it, if needed.

### CODE REVIEW

**Comment by tbarnes-us** [comment]
Yes, you are correct about my thoughts. In my opinion, the procedures are not duplicate and instead are convenient alternates for those that may want to edit the files.  Also, in the 'summary' edit, note that cutting and pasting is error prone and can be fairly tedious/problematic on some terminal setups.

**Review by robertpatrick — APPROVED** [review]

File: documentation/staging/content/quickstart/create-domain.md [hunk 1]
```diff
@@ -117,22 +115,13 @@ weight: 3
...
           port: 8001
     {{% /expand %}}
+
+    b. Then apply the file using:
     ```shell
     $ kubectl apply -f /tmp/quickstart/ingress-route.yaml \
       --namespace sample-domain1-ns
     ```
 
-      b. **NOTE**: Instead of running the previous  `kubectl apply` command, you can download the ingress route YAML file to `/tmp/quickstart/ingress-route.yaml` or similar, using the following command:
```
Comment (ankedia) [inline documentation/staging/content/quickstart/create-domain.md:125]: Based on slack discussion, should we provide a link to download the "ingress-route.yaml"? For e.g. you can download the ingress route YAML file from [here](https://raw.githubusercontent.com/oracle/weblogic-kubernetes-operator/main/kubernetes/samples/quick-start/ingress-route.yaml -o /tmp/quickstart/ingress-route.yaml)..

Comment (rosemarymarano) [reply documentation/staging/content/quickstart/create-domain.md:125]: Agree. Done.

**Review by ankedia — APPROVED** [review]
LGTM

**Pull request merged**